tests/src/Functional/ContentPersonRenderTest.php

```diff
@@ -0,0 +1,280 @@
+
+  use MediaTypeCreationTrait;
+  use TestFileCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'block',
+    'oe_whitelabel_starter_person',
+  ];
+
+  /**
+   * Tests the person page.
+   */
+  public function testPersonPage(): void {
+    $node = $this->createExamplePersonWithRequiredFieldsOnly();
+    $this->drupalGet($node->toUrl());
+    /** @var \Symfony\Component\BrowserKit\AbstractBrowser $client */
+    $client = $this->getSession()->getDriver()->getClient();
+    $crawler = $client->getCrawler();
+
+    // Assert required fields are present.
+    $content_banner = $crawler->filter('.bcl-content-banner');
+    $this->assertCount(1, $content_banner);
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $content_banner->filter('.card-title')->text()
+    );
+    $this->assertEquals(
+      'DG Test',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(2)')->text()
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert optional fields are not present.
+    // Assert no image is present.
+    $this->assertCount(0, $content_banner->filter('img'));
+
+    // Assert short description is not present.
+    $this->assertCount(0, $content_banner->filter('p'));
+
+    // Assert no social media links are present.
+    $this->assertCount(0, $content_banner->filter('a.standalone'));
+
+    $content = $crawler->filter('div.col-12.col-lg-10.col-xl-9.col-xxl-8');
+
+    // Assert additional information is not present.
+    $this->assertStringNotContainsString(
+      'Additional information',
+      $content->text()
+    );
+
+    // Assert documents are not present.
+    $this->assertStringNotContainsString(
+      'Related documents',
+      $content->text()
+    );
+
+    // Test person with all fields.
+    $node_with_all_fields = $this->createExamplePersonWithAllFields();
+    $this->drupalGet($node_with_all_fields->toUrl());
+    $crawler = $client->getCrawler();
+
+    $content_banner = $crawler->filter('.bcl-content-banner');
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $content_banner->filter('.card-title')->text()
+    );
+    $this->assertEquals(
+      'DG Test',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(2)')->text()
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert content banner image.
+    $image = $content_banner->filter('img');
+    $this->assertCount(1, $image);
+    $this->assertCount(1, $image->filter('.card-img-top'));
```

Comment:
For positive assertions we can start directly with the most specific selectors.
_Unless_ we want to prove that no additional images exist - but then we'd rather prove this for the entire page, not just the content banner.
```suggestion
    $image = $content_banner->filter('img.card-img-top');
    $this->assertCount(1, $image);
```